fire/cli/niv/ilæg_revision.py

```diff
@@ -85,9 +91,112 @@ def ilæg_revision(
         return
 
     revision = find_faneblad(f"{projektnavn}-revision", "Revision", ARKDEF_REVISION)
+    revision = revision.replace("nan", "")
     bemærkning = " ".join(bemærkning)
     opdateret = pd.DataFrame(columns=list(ARKDEF_REVISION))
 
+    # Udfyld udeladte identer
+    punkter = list(revision["Punkt"])
+    udfyldningsværdi = ""
+    for i in range(len(punkter)):
+        if punkter[i].strip() != "":
+            udfyldningsværdi = punkter[i].strip()
+            continue
+        punkter[i] = udfyldningsværdi
+    revision["Punkt"] = punkter
+
+    # Find alle punkter, der skal nyoprettes
+    oprettelse = revision.query(f"Attribut == 'OPRET'")
+    for row in oprettelse.to_dict("records"):
+        fire.cli.print(f"Opretter nyt punkt: {row['Punkt']}")
+        if alvor:
+            opret_punkt(row["Punkt"], row["Ny værdi"], sag)
+    revision = revision.query(f"Attribut != 'OPRET'")
+
+    # Find alle lokationskoordinater, der skal korrigeres
+    lokation = revision.query(f"Attribut == 'Lokation'")
+    lokation = lokation.query(f"`Ny værdi` != ''")
+    for row in lokation.to_dict("records"):
+        fire.cli.print(f"Korrigerer lokation for: {row['Punkt']}: {row['Ny værdi']}")
+        if alvor:
+            pass  # refaktor af opret_punkt(row["Punkt"], row["Ny værdi"], sag)
+    revision = revision.query(f"Attribut != 'Lokation'")
+
+    # Find alle koordinater, der skal oprettes
+
+    # Først skal vi bruge alle gyldige koordinatsystemnavne
+    srid_db = firedb.session.query(Srid).order_by(Srid.name).all()
+    sridnavne = [srid.name for srid in srid_db]
+
+    # Så looper vi over hele rammen og ignorerer ikke-koordinaterne
+    til_registrering = []
+    opdaterede_punkter = []
+    koordinatoprettelsestekst = str()
+    for r in revision.to_dict("records"):
+        if r["Attribut"].upper() not in sridnavne:
+            continue
+        try:
+            koord = [float(k) for k in r["Ny værdi"].split()]
+        except ValueError as ex:
+            fire.cli.print(
+                f"Slemt koordinatudtryk:\n{'    '.join(r['Ny værdi'])}\n{ex}"
+            )
+            sys.exit(1)
+
+        # Oversæt NaN til None
+        koord = [k if k == k else None for k in koord]
```

Comment:
Ikke et fuldbyrdet review, jeg studsede bare lige over den her. Efter man års PROJ arbejde har jeg lært at `NaN == NaN` evaluerer som False men det er dælme en uintuitiv konstruktion, så istedet foreslår jeg

```suggestion
        from math import isnan
        koord = [None if isnan(k) else k for k in koord]
```

som også er forståelig uden kendskab til NaNs opførsel i logiske udtryk.

Reply:
>  uintuitiv konstruktion

Nu vil jeg ikke håbe at der er nogen der fødes med forståelse af IEEE 754 NaN begrebet (det nærmeste relevante for spædbørn jeg er stødt på er modermælkserstatningsvaremærket NAN), så NaN er en tillært (dvs. tuitiv) egenskab hos os alle sammen.

Men det er altså den definerende egenskab ved NaN at en instans af *NaN ikke er lig med noget som helst - end ikke sig selv* (det er formodentlig også derfor `isnan()` først blev lukket ind i C standardbiblioteket med C99: Hvorfor gøre med et funktionskald hvad man kan gøre med en operator?).

Det skal nu ikke forhindre mig i at indføre `math.isnan()` - jeg vil bare ikke tages til indtægt for at det skulle gøre noget intuitivt forståeligt. Det er der meget lidt i symbolmanipulation der er.

Reply:
> jeg vil bare ikke tages til indtægt for at det skulle gøre noget intuitivt forståeligt. Det er der meget lidt i symbolmanipulation der er.

Det er så i orden, men så kan du da tages til indtægt for at gøre noget mere læseligt, især for det utrænede øje. Det må da være en skalp der er værd at få i bæltet :-)